README.md

```diff
@@ -1,54 +1,91 @@
 
-Should explain that every URL that we want our application to handle must map to a specific controller and action.
+Before we dive into the code and routing configurations, it helps to know how HTTP works at a high level. Below is the flow that takes place when a user attempts to go to a page on a Rails application:
 
-When a request comes into our application, our router must pick a controller and a method in that controller that is responsible for generating a response. We've taught a lot of this in Sinatra so it can be a bit light here.
+1. A URL is entered into the browser, this is the HTTP request
+2. That request is sent to the server where the application's router interprets the request and sends a message to the controller mapped to that route
+3. The controller communicates with the view file mapped to the controller method
+4. The server returns that HTTP response, which contains the view and page that can be viewed in the browser
 
-Use the example of how we want to have an 'about' page on our site. So let's try to go there, let's just type localhost:3000/about.
+## Implementing a Static Route
 
-We get a routing error. Let's draw the route.
+Let's try this out in our application, I'm going to continue using the ```blog-flash``` application that I walked through in the Rails Basic lesson.
 
-Let's draw a simple route, like /about. Let's have that map to a StaticController and an about action (though don't create that controller). Reload the page. No more routing error, now it can't find the controller.
+To begin, startup the rails server and go to ```localhost:3000/about``` as you will see this throws a routing error: ```No route matches [GET] "/about"```, to fix this, stop the rails server by pressing ```control + c``` (anytime you make a routing change you need to restart the rails server)
 
-Let's make a new controller, no generators, with touch app/controllers/static_controller.rb.
+Now draw the route by opening the ```config/routes.rb``` file and add the following route inside of the ```draw``` block:
 
-Controllers need to inherit from ApplicationController and explain briefly the controller naming convention.
+```ruby
+get 'about', to: 'static#about'
+````
 
-Define our first controller action as a method, def about.
+Now start the rails server back up and go back to ```localhost:3000/about``` and click refresh, you should now see that the error message has changed, it's no longer complaining about not having a route, the error should now say: ```uninitialized constant StaticController```
 
-Let's hit refresh. Now it complains about a view.
+Let's fix this by creating a new controller for our static pages, you can do this manually by running the following command in the terminal:
 
-Controller actions are responsible for determining what the response is - responses are built by views.
+```touch app/controllers/static_controller.rb```
```

Comment:
A lot use sublime text and such so don't want to confuse them with other commands. I've taught this before and students thought `touch` was a rais command haha